### PR TITLE
Fix Exception JSON

### PR DESCRIPTION
**Pull request by AjayShanker-geek**
Fix Exception caught on file creation

- Exception found creating empty json and loading it
- Fix logging format

### CODE REVIEW

**Review by NCF3535 — APPROVED** [review]
LGTM

File: src/main/java/seedu/duke/storage/JsonUtil.java [hunk 1]
```diff
@@ -27,37 +27,48 @@ public class JsonUtil {
             .enable(SerializationFeature.INDENT_OUTPUT) // Readable format // print
             .findAndRegisterModules(); // Automatically register additional modules (future use);
 
+    public static void saveToFile(String filePath) {
+        try {
+            objectMapper.writeValue(new File(filePath), new Hospital());
+        } catch (IOException e) {
+            // TODO: Update error handler (yes I am lazy)
```
Comment (NCF3535) [inline src/main/java/seedu/duke/storage/JsonUtil.java:34]: Try to implement this as soon as possible as saving files is one of our core features.
Perhaps this could work for the error handler:
logger.log(Level.SEVERE, "Failed to save data to file: {0}", e.getMessage());
throw new RuntimeException("Failed to save hospital data", e);

Comment (NCF3535) [inline src/main/java/seedu/duke/storage/JsonUtil.java:34]: Try to implement this as soon as possible as saving files is one of our core features.
Perhaps this could work for the error handler:
logger.log(Level.SEVERE, "Failed to save data to file: {0}", e.getMessage());
throw new RuntimeException("Failed to save hospital data", e);

File: src/main/java/seedu/duke/ui/Colors.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package seedu.duke.ui;
+
+/**
+ * Color class contains ANSI color codes for text color in the console.
+ */
+public class Colors {
```
Comment (NCF3535) [inline src/main/java/seedu/duke/ui/Colors.java:6]: Good that you created a class to handle ANSI color codes! It will help make the codes relating to ui neater.

File: src/test/java/seedu/duke/ui/UiTest.java [hunk 3]
```diff
@@ -17,14 +16,14 @@ class UiTest {
      */
     @Test
     public void readCommand_validInput_returnsCorrectCommand() {
```
Comment (NCF3535) [inline src/test/java/seedu/duke/ui/UiTest.java:18]: Maybe consider a test for invalid inputs will help us test edge cases.

File: text-ui-test/input.txt [hunk 4]
```diff
@@ -1,2 +1,2 @@
-James Gosling
+list
```
Comment (NCF3535) [inline text-ui-test/input.txt:1]: Maybe consider adding more input tests so that wwe can test all the functions of the program.

File: src/main/java/seedu/duke/storage/JsonUtil.java [hunk 5]
```diff
@@ -27,37 +27,48 @@ public class JsonUtil {
...
         }
-        logger.log(Level.INFO, "Data saved successfully at: " + filePath);
+        logger.log(Level.INFO, "Data saved successfully at: {0}", filePath);
     }
 
     public static Hospital loadFromFile(String filePath) {
         try {
             Hospital hospital = objectMapper.readValue(new File(filePath), Hospital.class);
 
             assert hospital != null : "Hospital object cannot be null";
```
Comment (NCF3535) [inline src/main/java/seedu/duke/storage/JsonUtil.java:56]: Exceptions may be more suitable here than assertions.

**Review by NCF3535 — APPROVED** [review]
LGTM, just some notes that we should take note for v2.0
